python/ray/data/_internal/arrow_block.py

```diff
@@ -554,22 +549,15 @@ def iter_groups() -> Iterator[Tuple[KeyType, Block]]:
                     break
 
         builder = ArrowBlockBuilder()
-        for group_key, group_view in iter_groups():
+        for group_keys, group_view in iter_groups():
             # Aggregate.
-            accumulators = [agg.init(group_key) for agg in aggs]
+            accumulators = [agg.init(group_keys) for agg in aggs]
```

Comment:
here, i believe `group_keys` will actually be a single key yielded from `iter_groups`. to avoid confusion, should we keep the name in singular form, `group_key`?

Reply:
OK, i addressed to make it more clear.